app/routes/overview/components/NextEvent.js

```diff
@@ -0,0 +1,126 @@
+// @flow
+
+import React from 'react';
+import { Link } from 'react-router';
+import styles from './NextEvent.css';
+import { colorForEvent } from 'app/routes/events/utils';
+import { Flex } from 'app/components/Layout';
+import type { Event } from 'app/models';
+import { Image } from 'app/components/Image';
+import alarm from 'app/assets/alarm.svg';
+import truncateString from 'app/utils/truncateString';
+import { orderBy } from 'lodash';
+import moment from 'moment-timezone';
+import Icon from 'app/components/Icon';
+import Tooltip from 'app/components/Tooltip';
+
+type Props = {
+  event: Event
+};
+
+class EventItem extends React.Component<Props, *> {
+  state = {
+    time: ''
+  };
+  interval: IntervalID;
+
+  setTime(props) {
+    moment.locale('no');
```

Comment:
I don't think you need this (but not 110% sure), I think we have a global one somewhere.